bin/loadtest.js

```diff
@@ -58,9 +59,12 @@ if (!options.args || options.args.length < 1) {
 	console.error('Too many arguments: %s', options.args);
 	help();
 }
+
+const configuration = config.loadConfig(options);
```

Comment:
Is this using a configurable path to the config file?

Reply:
For the moment, I use the default configuration file, as specified in [this documentation](https://github.com/slune-org/confinode/blob/HEAD/doc/en/usermanual.md#configuration-search). Simply using this enable to specify a configuration file in the `loadtest` entry of the `package.json` file.

But it would also be possible to add a configuration option, say `--config` in order to directly specify a configuration file.